tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -9,6 +9,21 @@ const downloadButtonSideBar = '#oc-files-actions-sidebar .oc-files-actions-downl
 const downloadButtonBatchActionSingleFile = '.oc-files-actions-download-file-trigger'
 const downloadButtonBatchActionMultiple = '.oc-files-actions-download-archive-trigger'
 const checkBox = `//*[@data-test-resource-name="%s"]//ancestor::tr//input`
+const fileRow = '//ancestor::tr'
+const resourceNameSelector = `[data-test-resource-name="%s"]`
+const addNewResourceButton = `#new-file-menu-btn`
+const createNewFolderButton = '#new-folder-btn'
+const folderNameInputField = '.oc-modal input'
+const resourceUploadButton = '#upload-menu-btn'
+const fileUploadInput = '#files-file-upload-input'
+const closeUploadInfoDialog = '#close-upload-info-btn'
+const filesAction = `.oc-files-actions-%s-trigger`
+const locationConfirmButton = '#location-picker-btn-confirm'
+const routerLinkActive = '//nav[contains(@class, "oc-breadcrumb")]/ol/li[1]/a'
```

Comment:
I didn't get what this selector is for. Maybe for `All files` in the breadcrumb?

Reply:
`filesAction` is for actions like `move/copy/rename` file/folder

Reply:
`filesAction` is for actions like `move/copy/rename` file/folder.
`locationConfirmation` is to confirm the location where a file/folder is going to be moved/copied.
`closeUploadInfoDialog` is a selector for file upload confirmation button.
`routerLinkActive` is for all files in the breadcrumb

Reply:
The comment is for the `routerLinkActive`.
If that one is for breadcrumb then it can be renamed to something like `breadcrumbRoot` or more understandable name